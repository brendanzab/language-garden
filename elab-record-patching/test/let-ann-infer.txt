let id : fun (A : Type) -> A -> A :=
  fun A a => a;

id {} {}
