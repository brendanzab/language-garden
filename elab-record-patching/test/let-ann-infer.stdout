<stdin> : {} :=
  let Id : Type := fun (A : Type) (_ : A) -> A;
  let id : fun (A : Type) (_ : A) -> A := fun (A : Type) (a : A) => a;
  id {} {}
